.travis.yml

```diff
@@ -18,7 +21,10 @@ before_script:
   - echo '{}' > "$HOME"/.docker/config.json
 
 script:
-  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration verify sonar:sonar -Dsonar.projectKey=com.exasol:row-level-security -Dsonar.login=${SONAR_TOKEN}
+  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration
+    verify sonar:sonar -Dsonar.login=${SONAR_TOKEN}
+    -Dsonar.coverage.jacoco.xmlReportPaths=target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml
```

Comment:
I think you need to add `-Dcom.exasol.dockerdb.image=${DOCKER_DB_VERSION}` as well if you wanna use the images defined above

Reply:
Done.